static/js/filters/date.js

```diff
@@ -29,13 +29,23 @@ var _ = require('underscore'),
     _.defaults(options, { prefix: '', suffix: '' });
 
     if (!date) {
-      return '<span>' + options.prefix + options.suffix + '</span>';
+      if (options.raw) {
```

Comment:
Why do we need a `raw` option? It looks like other dates in the live list use regular old dates.

Reply:
For the date to be added into the scope it does, otherwise the various HTML elements get escaped incorrectly. Putting it in the summary was the route I decided on after trying and failing to get it to work in HTML, due to issues with live list.

Reply:
Ahh right because you're running it through translation. I think you can change translation interpolation schemes to fix that but that'd leave the code open to injection I think. I think it's ok to leave it as is, I'm just wary of adding more options to an already cluttered API.